libs/wallet-ui/src/hooks/use-create-wallet.tsx

```diff
@@ -51,6 +51,7 @@ export function useCreateWallet() {
             type: 'ADD_WALLET',
             wallet: values.wallet,
             key: keypair,
+            auth: true,
           })
           dispatch({
             type: 'ACTIVATE_WALLET',
```

Comment:
How come this doesn't active the wallet correctly? Can this now be removed if auth: true is being set in the above action? 🤔

Reply:
yeah, good point, the activate here seems redundant, removed, thanks!